binding-library/java/src/main/java/com/microsoft/azure/functions/kafka/annotation/KafkaOutput.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.functions.kafka.annotation;
+
+import com.microsoft.azure.functions.annotation.CustomBinding;
+import com.microsoft.azure.functions.kafka.BrokerAuthenticationMode;
+import com.microsoft.azure.functions.kafka.BrokerProtocol;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.PARAMETER)
+@Retention(RetentionPolicy.RUNTIME)
+@CustomBinding(direction = "in", name = "", type = "KafkaOutput")
+public @interface KafkaOutput { // TODO Should I name it as KafkaOutput?
+    /**
+     * Gets the Topic.
+     * @return
+     */
+    String topic();
+
+    /**
+     * Gets or sets the BrokerList.
+     */
+    String brokerList();
+
+    /**
+     * Gets or sets the KeyType
+     * This method is used internally. Don't pass the value to this method.
+     */
+    // String keyType();  // TODO Originally Type type. Should I pass the serialized value for them?
+
+    /**
+     * Gets or sets the ValueType
+     * This method is used internally. Don't pass the value to this method.
+     */
+    // String valueType(); // TODO Originally Type type. Should I pass the serialized value for them?
+
+    /**
+     * Gets or sets the Avro schema.
+     * Json format
+     * Default: Plain*
+     */
+    String AvroScema();
```

Comment:
I am not sure it makes sense to expose this property. They work well when serializing to a C# object. I don't know if we can translate the POCO to POJO.

Reply:
Yes. POJO works!

Reply:
Sure. I remove it.

Reply:
Thanks @pragnagopa, was not aware. Where would the serialization happen
(Pojo -> byte[])?

Reply:
For each non C# language we support, there is a language worker handles function invocations. For example for java, there is a https://github.com/Azure/azure-functions-java-worker/ . It is language worker's responsibility to convert input to the requested type.  In this case specifically, here is the code that converts to POJO: [DataOperations.java](https://github.com/Azure/azure-functions-java-worker/blob/a690cdd30850a321eb794f19fce067c9e4fe97a1/src/main/java/com/microsoft/azure/functions/worker/binding/DataOperations.java#L83-L123)